projects/sonar/src/app/_layout/admin/admin.component.ts

```diff
@@ -112,5 +114,17 @@ export class AdminComponent implements OnInit, OnDestroy {
    */
   changeLanguage(languageCode: string) {
     this._translateService.setLanguage(languageCode);
+    this._changeFlaskLanguage(languageCode);
+  }
+
+  /**
+   * Change language on frontend flask.
+   *
+   * @param languageCode 2 digit language code.
+   */
+  private _changeFlaskLanguage(languageCode: string): void {
+    this._httpClient
+      .get(`/lang/${languageCode}`, { responseType: 'text' })
```

Comment:
A `GET` method seems not really clean, but maybe it is related the to backend API.

Reply:
Exactly. It depends on the backend